test/src/test/java/hudson/slaves/SlaveComputerTest.java

```diff
@@ -0,0 +1,19 @@
+package hudson.slaves;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class SlaveComputerTest {
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void testGetAbsoluteRemotePath() throws Exception {
```

Comment:
I doubt this test actually tests anything, but works as a smoke test

Reply:
I want put some permission test case, but haven't figure it out who to do it.